### PR TITLE
feat(pgw): document ip mobility

### PR DESCRIPTION
**Pull request by RoRoJ**
Document IP mobility for Public Gateways

image
image

image

### CODE REVIEW

File: network/public-gateways/how-to/move-to-routed-ip.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+- There should be only a few seconds of service interruption
+- Your IP address will not change and no additional configuration should be necessary
+- Once done, this action cannot be undone
+- Via the Scaleway console, you can only move Public Gateways one at a time to routed IP
+- Via the [API](https://www.scaleway.com/en/developers/api/vpc/), you can move multiple Public Gateways at a time to routed IP.
+
+Moving your Public Gateway to routed IPs will ensure that when features such as IPv6 are released in the future, your gateway will be compatible. The move from NAT to routed IPs is part of Scaleway's overarching IP mobility project. Read our blog post for [full information](https://www.scaleway.com/en/blog/ip-mobility-removing-nat/).
+
+
+<Macro id="iam-requirements" />
```
Comment (jcirinosclwy) [inline network/public-gateways/how-to/move-to-routed-ip.mdx:32]:
```suggestion
<Macro id="requirements" />
```

File: network/public-gateways/how-to/move-to-routed-ip.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Note the following when moving a Public Gateway from NAT IP to routed IP:
+- There should be only a few seconds of service interruption
+- Your IP address will not change and no additional configuration should be necessary
+- Once done, this action cannot be undone
+- Via the Scaleway console, you can only move Public Gateways one at a time to routed IP
+- Via the [API](https://www.scaleway.com/en/developers/api/vpc/), you can move multiple Public Gateways at a time to routed IP.
+
+Moving your Public Gateway to routed IPs will ensure that when features such as IPv6 are released in the future, your gateway will be compatible. The move from NAT to routed IPs is part of Scaleway's overarching IP mobility project. Read our blog post for [full information](https://www.scaleway.com/en/blog/ip-mobility-removing-nat/).
+
+
```
Comment (jcirinosclwy) [inline network/public-gateways/how-to/move-to-routed-ip.mdx:31]:
```suggestion
```

File: network/public-gateways/how-to/move-to-routed-ip.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
```
Comment (jcirinosclwy) [inline network/public-gateways/how-to/move-to-routed-ip.mdx:34]:
```suggestion
```

File: network/public-gateways/how-to/move-to-routed-ip.mdx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```
Comment (jcirinosclwy) [inline network/public-gateways/how-to/move-to-routed-ip.mdx:35]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com)
```

File: network/public-gateways/how-to/move-to-routed-ip.mdx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  - You have a Public Gateway created before [TODODATE] which is not yet moved to routed IP
```
Comment (jcirinosclwy) [inline network/public-gateways/how-to/move-to-routed-ip.mdx:36]:
```suggestion
- A Public Gateway created before [TODODATE] which is not yet moved to routed IP
```

File: network/public-gateways/how-to/move-to-routed-ip.mdx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+</Message>
```
Comment (jcirinosclwy) [inline network/public-gateways/how-to/move-to-routed-ip.mdx:37]:
```suggestion
```

File: network/public-gateways/how-to/move-to-routed-ip.mdx [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+You can move your Public Gateway from NAT to routed IP addressing by following the simple steps outlined on this page. **Your IP address itself will not change**. This is purely a backend modification to the method Scaleway uses to map IP addresses to the physical hypervisors hosting your resources. Moving to routed IP will simply update your current network interface for a new one, while keeping your attached IP address.
+
+In this first phase, you may choose when to move your Public Gateway from NAT to routed by following the steps below at a time best suited to you. In a future phase, moving to routed IP will be done automatically for all Public Gateways that still have NAT IPs. The timing of this will be announced in future communications. We strongly recommend that you move your gateway to routed IP yourself, before automatic migration.
+
+Note the following when moving a Public Gateway from NAT IP to routed IP:
+- There should be only a few seconds of service interruption
+- Your IP address will not change and no additional configuration should be necessary
```
Comment (jcirinosclwy) [inline network/public-gateways/how-to/move-to-routed-ip.mdx:24]:
```suggestion
- Your IP address will not change, and no additional configuration should be necessary
```

**Review by jcirinosclwy — APPROVED** [review]

File: network/public-gateways/how-to/move-to-routed-ip.mdx [hunk 8]
```diff
@@ -0,0 +1,76 @@
+---
+meta:
+  title: How to move your Public Gateway to a routed IP
+  description: This page explains how to move your Public Gateway from a legacy NAT IP addressing solution, to a routed IP solution, compatible with Scaleway's IP mobility project and features.
+content:
+  h1: How to move your Public Gateway to a routed IP
+  paragraph: This page explains how to move your Public Gateway from a legacy NAT IP addressing solution, to a routed IP solution, compatible with Scaleway's IP mobility project and features.
+tags: public-gateway public gateway nat-ip routed-ip ip-mobility
+dates:
+  validation: 2024-02-21
```
Comment (nerda-codes) [inline network/public-gateways/how-to/move-to-routed-ip.mdx:10]: maybe change dates?

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**